dvc/utils/fs.py

```diff
@@ -202,6 +198,12 @@ def copyfile(src, dest, no_progress_bar=False, name=None):
                     fdest_wrapped.write(buf)
 
 
+def copy_fobj_to_file(fsrc, dest):
```

Comment:
I think we will probably want progress bar support for this once we are saving files from git tree to local cache, but I think it may make more sense to have the pbar instance in the higher level `save()` method.

So at that point we can add a pbar update callback parameter to this method (and do buffered read/write instead of using `shutil.copyfileobj` as needed).